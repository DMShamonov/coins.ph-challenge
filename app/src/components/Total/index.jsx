// @flow

import * as React from 'react';

import type { PropsType } from './types.js.flow';

export default function Total({
  value,
}: PropsType): React.Node {
  return (
    <p className="total">{`${value} Total`}</p>
  );
}
