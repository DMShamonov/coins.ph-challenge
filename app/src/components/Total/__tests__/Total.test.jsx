import React from 'react';
import { shallow } from 'enzyme';

import Total from '../index';

describe('`Total` component should render correctly', () => {
  test('with default props', () => {
    expect(shallow(<Total value={10} />)).toMatchSnapshot();
  });
});
