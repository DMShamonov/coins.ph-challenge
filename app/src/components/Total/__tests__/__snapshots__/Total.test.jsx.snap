// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`\`Total\` component should render correctly with default props 1`] = `
<p
  className="total"
>
  10 Total
</p>
`;
